Maestro/subscriptions.json

```diff
@@ -623,6 +651,31 @@
         }
       }
     },
+    // Update dependencies in CoreFX release/2.2
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/product/cli/release/2.2/packages.semaphore"
```

Comment:
I think the Standard and BuildTools triggers should still be here, since they're still listed in dependencies.props and auto-update will still attempt to make updates whenever this does happen to get triggered.

I haven't thought about how to reconcile all this with the auto-merges from 2.1 -> 2.2. 😕 If conflict resolution is really good, then the triggers don't seem necessary. (And any happenstance upgrades that get merged into both branch independently will be resolved.)

No disaster either way, though, feel free to merge as-is.

Reply:
Yeah that's what I was thinking..but may as well leave them in.